gpkit/constraints/tight.py

```diff
@@ -35,7 +35,7 @@ def process_result(self, result):
                         leftsubbed = constraint.left.sub(variables).value
                         rightsubbed = constraint.right.sub(variables).value
             if rel_diff >= self.reltol:
-                msg = ("Constraint [%.30s...] is not tight because "
+                msg = ("Constraint [%.200s...] is not tight because "
                        "the left hand side evaluated to %s but "
```

Comment:
printing a certain number of left and right variables is probably the best solution, how do you print just a variable and not the varkey?